test/extended/clusterup.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
```

Comment:
yes, I would like to run this extended job on PRs with the default extended test annotation for rosiebot